package.json

```diff
@@ -72,9 +71,9 @@
     "remark-parse": "^11.0.0",
     "remark-rehype": "^11.1.2",
     "remark-stringify": "^11.0.0",
-    "rolldown": "^1.0.0-beta.27",
-    "semver": "^7.7.1",
-    "shiki": "^3.7.0",
+    "rolldown": "^1.0.0-beta.28",
+    "semver": "^7.7.2",
```

Comment:
I think https://github.com/vltpkg/vltpkg/tree/main/src/semver it's better and same api

Reply:
https://bundlephobia.com/package/semver@7.7.2 vs https://bundlephobia.com/package/@vltpkg/semver@0.0.0-17

SGTM

Reply:
https://github.com/vltpkg/vltpkg/tree/main/src/semver#differences-from-node-semver-the-one-used-by-npm

Reply:
okay 👍